packages/tailwind-ui/tailwind.css

```diff
@@ -1,4 +1,15 @@
 @theme {
+  --color-primary-50: #F4F6FB;
```

Comment:
I have a couple of thoughts about color naming conventions, but definitely open to other opinions:

1. Tailwind uses a color descriptor (e.g. `color-blue-500`, `color-lime-50`) when defining each of the steps for a particular color. 
2. I typically see `color-primary` defined as a single color. I could see that as an advantage not needing to define 10 color steps for the primary color.

Reply:
These values come from the theme file in Figma. I'm not opposed to using slightly different names for the colors but they also need to be easy to correlate with the names on the board. This Figma board obscures colors behind variable names more than usual, I guess because it's using a custom color scheme.

Reply:
Do we know how the colors are named in Figma? Is it done automatically? I just don't want us to go down a sub-optimal development path for the sake of design software.

Reply:
> From a cursory scroll around the board, I only see primary-600 being used. We might be able to drop the others and turn primary-600 into just primary.

Alright, I've done that for now. Re-requesting review.